pkg/manager/risingwave_controller_manager.cm

```diff
@@ -55,7 +55,7 @@ decl RisingWaveControllerManager for RisingWave {
 
         // Service for standalone node.
         standaloneService Service {
-            name=${target.Name}
+            name=${target.Name}-standalone
```

Comment:
Why do we need to suffix `-standalone`?

Reply:
Because we added it in the last PR #519. This fixes a mismatch between factory method and getter.

Reply:
To me it seems like there should be some common interface between factory method and getter to avoid having to suffix this string. Seems rather error prone.